utils/modify-date.js

```diff
@@ -0,0 +1,16 @@
+const _ = undefined;
```

Comment:
Why is here such a strange constant? Is it exists only for test above? If that so than why don't you just write modifyNowDate(null, null, 10) for example?

Reply:
Not just for tests. This function can be reused in other parts of code. There is a choice, write modifyNowDate( undefined, undefined, 10 ) or shortener - modifyNowDate(  _, _, 10 ) . I don't know which way is better.